src/utils/domain.ts

```diff
@@ -1,24 +1,34 @@
-import { ScaleLinear, ScaleBand } from 'd3-scale';
-import { extent } from 'd3-array';
-import { IGroupDataItem } from '../HistogramD3';
-import { EGroupedBarLayout, IDomain, IAxes, IAxis, IHistogramDataSet } from '../Histogram';
 import { Axis } from 'd3';
-import get from 'lodash/get';
-import { timeFormat } from 'd3-time-format';
+import { extent } from 'd3-array';
 import { format } from 'd3-format';
+import { ScaleLinear } from 'd3-scale';
+import { timeFormat } from 'd3-time-format';
+import get from 'lodash/get';
 
-export const  isStacked = ({ groupLayout, stacked }) => {
+import {
+  EGroupedBarLayout,
+  IAxis,
+  IDomain,
+  IHistogramDataSet,
+} from '../Histogram';
+import { IGroupData } from '../HistogramD3';
+import { AnyScale } from './scales';
+
+export const isStacked = ({ groupLayout, stacked }) => {
```

Comment:
as in `bar.ts`, are there types available for this object?

Reply:
added! :)